### ISSUE TITLE
merge option: It's not possible to use a merged report as input

### ISSUE THREAD

**Issue by SimonKagstrom** [SimonKagstrom/kcov#119]
Kcov cannot merge merged output, i.e., running

```
kcov /tmp/a <program>
kcov /tmp/b <other program>

kcov --merge /tmp/merged /tmp/a /tmp/b
```

produces a merged report in /tmp/merged. But using this as an input file, e.g.,

```
kcov /tmp/c <third program>
kcov --merge /tmp/second-merge /tmp/merged /tmp/c
```

only produces the coverage from /tmp/c.

The reason is that the merge output doesn't create a coverage database. It should not be overly difficult to add this.

**Comment by SimonKagstrom** [comment]
Apparently trickier than I thought, but with the commits above a coverage database is created and filled for the merge-mode as well. However, while the files in the merge reports show up, they get zero coverage.

**Comment by SimonKagstrom** [comment]
Fixed with f2328b5.

The final problem was that the merge-parser incorrectly reported hits with the line number instead of the address.
